Impostor.Hazel.Abstractions/MessageType.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Impostor.Hazel.Abstractions;
+
+/// <summary>
+///     Specifies how a message should be sent between connections.
+/// </summary>
+[Flags]
```

Comment:
Why is this [Flags] btw? It was already like this in the old version, but having Unreliable and Reliable at the same time doesn't make sense

Reply:
Good question, [not for me though](https://github.com/DarkRiftNetworking/Hazel-Networking/blob/master/Hazel/SendOption.cs#L11). I guess we could remove it?